docs/source/developer/architecture/backend/data/GroupingMap.rst

```diff
@@ -0,0 +1,72 @@
+.. _grouping-map-design:
+
+================================================
+Grouping-schema map (as ``GroupingMap`` object):
+================================================
+
+*********
+Overview:
+*********
+
+Each detector on the instrument is comprised of pixels, which are the smallest sensor units.  These pixels are identified by their indices (global to the instrument), which are colloquially referred to as <detector id>s.
+
+During the first stages of processing data from a specified run, data from groups of pixels are combined and processed together.  A grouping schema is used to assign pixels to groups, thereby associating each detector id with a group number.  Such a schema is most commonly represented in Mantid as a grouping workspace.  The specific representation of the grouping workspace is as a map between detector-id and group number, implemented as a vector of consecutive spectra, where each spectrum number is the detector-id, and each single value is the corresponding group number.
+
+Depending on the requirements of data processing, several possible grouping schema may be used. Examples of available grouping schemas are, "all": a single group including all pixels in the instrument; "column": each group includes pixels belonging to a column of detectors; "bank": each group includes pixels belonging to a bank of detectors.
```

Comment:
Could you add entries to the `glossary.rst` and link them throughout the doc with the 
```
:term:`Your Domain Word`
```
specifier?

Reply:
I added extensive links to the glossary, as well as extending the glossary.  Some of my entries will possibly need some work, but they may be OK for the interim.